cygnus-ngsi/src/test/java/com/telefonica/iot/cygnus/sinks/NGSIMySQLSinkTest.java

```diff
@@ -32,7 +32,7 @@
  * @author frb
  */
 public class NGSIMySQLSinkTest {
-    
```

Comment:
Not real changes in NGSIMySQLSinkTest.java file, just indent-fixing, right? (Just checking)

Reply:
Moreover, looking to original issue:

> The scope for this issue only considers Postgres and Postgis, this because it's necessary to make some code cleanning for MySQL Sink before this task can be done on it. (issue #1783)

So, if at the end no tchange has been donde in the SQL Sink, I think that cleanining is not required at the end to progress this PR (and it would be done later in the context of issue #1783). Can you clarify, please?

Reply:
I actually modified NGSIMySQLSinkTest and NGSIMySQLSink but since changes were a bit complicated we (@manucarrace and me) decided that NGSIMySQLSink was going to have it's own issue since the code needed some cleaning before this issue can be solved. 

So when I reset those commits for some reason when I forced pushed again git marked thos blank spaces as changes, maybe because for compress purposes git removes blank spaces.

Reply:
Ok, crystal clear :) NTC